samples/core/SampleApps/SimpleWindowsFormsApp/Category.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.EntityFrameworkCore.ChangeTracking;
+
+namespace SimpleWindowsFormsApp
+{
+    public class Category
+    {
+        private readonly ObservableCollectionListSource<Product> _products =
+                new ObservableCollectionListSource<Product>();
+
+        public int CategoryId { get; set; }
+        public string Name { get; set; }
+        public virtual ObservableCollectionListSource<Product> Products { get { return _products; } }
```

Comment:
Could use an auto property:
```cs
Products { get; } = new ObservableCollectionListSource<Product>()
```

Reply:
Yeah. I was going to do this in another PR. I wanted to change as little as possible from the EF6 version for the initial checkin.